doc/dev/README.md

```diff
@@ -3,13 +3,15 @@
 A quick start guide to get Kepler up and running inside your container-based development cluster.
 
 ## Create a new emphemeral local kubernetes cluster
-Bring the cluster up with:
+Bring the cluster up with following [repo](https://github.com/sustainable-computing-io/KeplerK8SAction)
```

Comment:
The instructions here could be rephrased a bit for clarity. Which steps should the user take?
If I understand correctly, the action is meant to be used as part of a GitHub Actions workflow, so the title of this section on L5 could be changed to reflect that too :)

Reply:
no, once up on the time. @rootfs suggested as we have a local set up for development env.
Hence, I don't want to duplicate document here @nikimanoledaki which means:

Option A
```
document details under GitHub Action repo.
Other repo 
 as kepler points document to GitHub Action repo
 as kepler helm points document to GitHub Action repo
 as kepler operator points document to GitHub Action repo
```

Option B
```
GitHub Action repo has its own document.
kepler has its own document.
kepler helm...
kepler operator ...
```

Hence here points to KeplerK8SAction repo

Reply:
Before resolving it, would it be possible to rename it to `kepler-action`? :)
```suggestion
Bring the cluster up with following [repo](https://github.com/sustainable-computing-io/kepler-action)
```

Reply:
This is the new name, right?